### ISSUE TITLE
Default text color is orange, should be white

### ISSUE THREAD

**Issue by smlombardi** [Microsoft/vscode#20141]
- VSCode Version: Code - Insiders 1.10.0-insider (87fd1dde6d2d2b0d84c3fe56fabc174e888b0024, 2017-02-07T07:05:01.586Z)
- OS Version: Darwin x64 16.4.0
- Extensions:

|Extension|Author|Version|
|---|---|---|
|code-settings-sync|Shan|2.4.3|
|sort-lines|Tyriar|1.2.0|
|html-css-class-completion|Zignd|1.0.3|
|Bookmarks|alefragnani|0.10.2|
|project-manager|alefragnani|0.13.2|
|vscode-angular-files|alexiv|1.3.7|
|vscode-eslint|dbaeumer|1.2.4|
|tslint|eg2|0.8.1|
|vscode-great-icons|emmanuelbeziat|1.1.44|
|auto-rename-tag|formulahendry|0.0.11|
|Angular2|johnpapa|1.0.2|
|vscode-icon-theme|jtlowe|1.4.0|
|theme-karyfoundation-themes|karyfoundation|11.0.0|
|ftp-sync|lukasz-wronski|0.3.2|
|HTMLHint|mkaufman|0.3.3|
|vscode-autoprefixer|mrmlnc|2.0.0|
|vscode-csscomb|mrmlnc|4.0.0|
|vscode-postcss-sorting|mrmlnc|2.2.1|
|vscode-stylefmt|mrmlnc|2.2.2|
|Theme-1337|ms-vscode|0.1.2|
|angular2-inline|natewallace|0.0.17|
|vscode-icons|robertohuertasm|7.2.0|
|stylelint|shinnn|0.22.0|
|darcula-extended|smlombardi|3.3.2|
|slime|smlombardi|1.0.2|
|theme-espresso|smlombardi|1.0.1|
|theme-tesla|smlombardi|6.0.0|
|change-case|wmaurer|1.0.0|

---

Steps to Reproduce:

1. Create a new file and paste some text in it
2. The text is orange, when I'd expect it to be white. My theme has the generic foreground color as white.  I am not using that orange anywhere in the theme.

![screen shot 2017-02-07 at 1 01 28 pm](https://cloud.githubusercontent.com/assets/835148/22704452/c4352ada-ed35-11e6-9510-b5ec01e5660e.png)

**Comment by Tyriar** [comment]
This is probably a theme problem, have you published your theme or do you have a link to it?

**Comment by smlombardi** [comment]
https://marketplace.visualstudio.com/items?itemName=smlombardi.slime

If it's a theme problem I'd love to know what it is so I can fix.

**Comment by Tyriar** [comment]
This may be a bug as this is the only rule that uses that color:

```xml
            <dict>
                <key>name</key>
                <string>CSS @rule keyword control</string>
                <key>scope</key>
                <string>
                    meta.at-rule.return.scss, 
                    meta.at-rule.return.scss punctuation.definition,                    
                    meta.at-rule.else.scss, 
                    meta.at-rule.else.scss punctuation.definition,
                    meta.at-rule.if.scss, 
                    meta.at-rule.if.scss punctuation.definition,
                </string>
                <key>settings</key>
                <dict>
                    <key>foreground</key>
                    <string>#CC7832</string>
                </dict>
            </dict>
```

@alexandrudima please investigate :smiley:

**Comment by smlombardi** [comment]
I cannot fix this on my end.  I have tried some high-level selectors:
```xml
<dict>
                <key>name</key>
                <string>Text</string>
                <key>scope</key>
                <string text, source</string>
                <key>settings</key>
                <dict>
                    <key>foreground</key>
                    <string>#EFEFEF</string>
                </dict>
            </dict>
```
But there's no way to fix a "plain text" file, and all syntaxes flash orange for a second before the actual syntax colors appear.

**Comment by Tyriar** [comment]
The impact of this change may be larger than initially thought https://github.com/gerane/VSCodeThemes/issues/33, breaking the default color in many themes. @alexandrudima consider fixing for recovery build if there's time.

**Comment by alexandrudima** [comment]
Fixed in Microsoft/vscode-textmate#35 and adopted vscode-textmate@3.1.2 with 24f219194adc8a098a3065dde87f64842ce5ac88
